### PR TITLE
Prepare 1.0.0 release

### PR DESCRIPTION
**Pull request by arendjr**

### CODE REVIEW

**Review by mies — APPROVED** [review]

File: fp-bindgen/src/generators/rust_plugin/mod.rs [hunk 1]
```diff
@@ -62,12 +62,12 @@ fn generate_cargo_file(
                 git: Some("ssh://git@github.com/fiberplane/fp-bindgen.git"),
                 branch: Some("main"),
                 path: None,
-                version: None,
+                version: Some("1.0.0-rc.1"),
```
Comment (hatchan) [inline fp-bindgen/src/generators/rust_plugin/mod.rs:65]: We could replace this with `env!("CARGO_PKG_VERSION")` so it will use the same version.

**Review by Zagitta — CHANGES_REQUESTED** [review]

File: fp-bindgen-support/Cargo.toml [hunk 2]
```diff
@@ -1,10 +1,16 @@
 [package]
 name = "fp-bindgen-support"
-version = "0.1.0"
+description = "Support crate for fp-bindgen"
+homepage = "https://github.com/fiberplane/fp-bindgen/"
+repository = "https://github.com/fiberplane/fp-bindgen/"
+readme = "../README.md"
```
[the remark this answers is not in the document]
Comment (arendjr) [reply fp-bindgen-support/Cargo.toml:6]: It does indeed not work :( Think I'll solve it with a symlink then.

File: fp-bindgen/src/generators/rust_plugin/mod.rs [hunk 3]
```diff
@@ -62,12 +62,12 @@ fn generate_cargo_file(
                 git: Some("ssh://git@github.com/fiberplane/fp-bindgen.git"),
```
[the remark this answers is not in the document]
Comment (arendjr) [reply fp-bindgen/src/generators/rust_plugin/mod.rs:62]: You mean the crate path with the repo? Cargo will see it's a workspace and then figure out the correct package to use based on the name.

[the remark this answers is not in the document]
Comment (Zagitta) [reply fp-bindgen/src/generators/rust_plugin/mod.rs:62]: No I mean this is the generated `Cargo.toml` for the bindings crate which should point to crates.io for `fp-bindgen-support` right?

[the remark this answers is not in the document]
Comment (arendjr) [reply fp-bindgen/src/generators/rust_plugin/mod.rs:62]: But that's what it does now, thanks to it specifying the version, right?

[the remark this answers is not in the document]
Comment (Zagitta) [reply fp-bindgen/src/generators/rust_plugin/mod.rs:62]: Oh okay right, I wasn't aware the version number had priority over the url & branch. Would it make more sense to change url & branch to `None` instead to make it less confusing then?

[the remark this answers is not in the document]
Comment (arendjr) [reply fp-bindgen/src/generators/rust_plugin/mod.rs:62]: Yeah, that makes sense indeed :+1:

**Review by Zagitta — APPROVED** [review]

**Pull request merged**